WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
Adding another class / more methdos, honestly, sounds a bit of an overkill to me!.  But your call @koke, that'd be a design decision (i think as long as it can match Calypso's standard UI, it'd be okay).